### PR TITLE
fix(platform): Refresh doc setup instruction

### PR DESCRIPTION
**Pull request by majdyz**
### Background

The development setup has changed a lot, and some of the instructions is no longer valid.

### Changes 🏗️

* Update setup instruction
* Simplify the prerequisite by not forcing `yarn`.
* Reduce explicit `cd`s and reduce the amount.

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

**Comment by aarushik93** [comment]
cc @Bentlybro

**Comment by Bentlybro** [comment]
Thanks for the tag @aarushik93 ill go over this a look at this tomorrow

**Comment by ntindle** [comment]
Consider looking into this https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks-c for yarn/npm

File: autogpt_platform/README.md [hunk 1]
```diff
@@ -8,39 +8,60 @@ Welcome to the AutoGPT Platform - a powerful system for creating and running AI
...
+   ```
+   cp supabase/docker/.env.example .env
+   ```
+   This command will copy the `.env.example` file to `.env` in the `supabase/docker` directory. You can modify the `.env` file to add your own environment variables.
+
+4. Run the following command:
    ```
    docker compose up -d
+   ```
+   This command will start all the necessary backend services defined in the `docker-compose.combined.yml` file in detached mode.
```
Comment (Swiftyos) [inline autogpt_platform/README.md]: This has should be `docker-compose.yml`

File: docs/content/server/setup.md [hunk 2]
```diff
@@ -19,121 +19,99 @@ We also offer this in video format. You can check it out [here](https://github.c
...
-   ```sh
-   docker compose up postgres redis -d
-   poetry run prisma migrate dev
-   ```
-After deploying the migration, to ensure that the database schema is correctly mapped to your codebase, allowing the application to interact with the database properly, you need to generate the Prisma database model:
+* Run the backend services:
+  ```
+   docker compose up -d
+  ```
+  This command will start all the necessary backend services defined in the `docker-compose.combined.yml` file in detached mode.
```
Comment (Swiftyos) [inline docs/content/server/setup.md:83]: This should just be `docker-compose.yml` now

File: docs/content/server/setup.md [hunk 3]
```diff
@@ -19,121 +19,99 @@ We also offer this in video format. You can check it out [here](https://github.c
...
 ### Notes: 
-By default the daemons for different services run on the following ports: 
+By default the appplication for different services run on the following ports: 
 
-Execution Manager Daemon: 8002
-Execution Scheduler Daemon: 8003
-Rest Server Daemon: 8004
+Frontend UI Server: 3000
+Backend Websocket Server: 8001
+Backend API Rest Server: 8006
```
Comment (Swiftyos) [inline docs/content/server/setup.md]: Marketplace API Server: 8015

Also

Prisma studio can be accessed from `http://127.0.0.1:8000` Using the default login credentials:

```
DASHBOARD_USERNAME=supabase
DASHBOARD_PASSWORD=this_password_is_insecure_and_should_be_updated
```

**Review by Swiftyos — CHANGES_REQUESTED** [review]

File: docs/content/server/setup.md (hunk 2 above)
Comment (majdyz) [reply docs/content/server/setup.md:83]: Updated

File: autogpt_platform/README.md [hunk 4]
```diff
@@ -8,40 +8,61 @@ Welcome to the AutoGPT Platform - a powerful system for creating and running AI
...
 ### Running the System
 
 To run the AutoGPT Platform, follow these steps:
 
-1. Clone this repository to your local machine.
-2. Navigate to autogpt_platform/supabase
-3. Run the following command:
+1. Clone this repository to your local machine and navigate to the `autogpt_platform` directory within the repository:
+   ```
+   git clone <https://github.com/Significant-Gravitas/AutoGPT.git | git@github.com:Significant-Gravitas/AutoGPT.git>
```
Comment (ntindle) [inline autogpt_platform/README.md:19]: Does this syntax actually work if copied

File: docs/content/server/setup.md (hunk 3 above)
Comment (majdyz) [reply docs/content/server/setup.md]: Marketplace API should be eventually replaced with the cloud version and it doesn't make sense to use the the local version of it.

Also the one running on `http://127.0.0.1:8000` is supabase studio not prisma, and currently it's also not working.

File: autogpt_platform/README.md (hunk 4 above)
Comment (majdyz) [reply autogpt_platform/README.md:19]: Well it works but you had to choose one of them.

File: autogpt_platform/README.md (hunk 1 above)
Comment (majdyz) [reply autogpt_platform/README.md]: Updated

**Comment by majdyz** [comment]
> Consider looking into this https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks-c for yarn/npm

I removed entirely the occurrence of yarn on mkdocs documentation, the one remain is the readme.md on the repository.

**Review by ntindle — APPROVED** [review]

**Pull request merged**